packages/CollapsibleCard/src/components/Header/Header.styles.js

```diff
@@ -21,27 +23,27 @@ export const Header = styled.div(
           box-shadow: inset 0px -1px 0px ${tokens.color.blackLighten60};
         `}
 
-      margin: 0px;
+      margin: 0;
       position: relative;
     }
 
     ${isEditing &&
       css`
         background-color: ${tokens.color.yellowLighten30};
 
-        :hover {
+        &:hover {
           background-color: #ffebc2;
         }
       `}
 
     &:focus {
-      outline: none;
-      position: relative;
+      ${stylers.focusRing()}
+      &:not(:focus-visible) {
+        box-shadow: none;
+      }
```

Comment:
since `:focus & &:not(:focus-visible)` has been used number of different places.. feel like we should just have a mixin for it ?